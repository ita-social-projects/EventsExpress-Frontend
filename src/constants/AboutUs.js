const ABOUT_US = {
  JOIN_US: "Join us",
  ABOUT_US: "About us",
  CONTACT_US: "Contact us",
  SEARCH_EVENTS: "Search events",
  CREATE_YOUR_EVENT: "Create your event",
  FIND_EVENTS: "Find events",
  GET_CONNECTED: "Get connected",
  HAVE_FUN_TOGETHER: "Have fun together",
  LOREM_IPSUM:
    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim adminim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt inculpa qui officia deserunt mollit anim id est laborum. ",
};

export default ABOUT_US;
