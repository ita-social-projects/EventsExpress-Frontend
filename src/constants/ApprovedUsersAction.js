const ApprovedUsersActions = {
  WANNA_APPROVE: " Are you sure, that you wanna approve ",
  TO_OWNER: " to owner? ",
  DELETE_FROM_EVENT: " Delete from event ",
};

export default ApprovedUsersActions;
