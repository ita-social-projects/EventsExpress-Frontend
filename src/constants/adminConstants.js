export const CONTACT_ADMIN_CONSTS = {
  TITLE: "Title",
  DATE_CREATED: "Date created",
  STATUS: "Status",
  DETAILS: "Details",
  ISSUE_DESC: "ssue description",
};

export const CONTACT_ADMIN_COMP_CONSTS = {
  CONTACT_US: "Contact Us",
  PROBLEM_TYPE: "Problem type",
  NEW_CATEGORY: "New Category",
  BUG_REPORT: "Bug Report",
  BAD_EVENT: "Bad Event",
  BAD_USER: "Bad user",
  OTHER: "Other",
};

export const EMPTY_ISSUES_ARRAY = 0;
export const ADMIN_MIN_DATE_FILTER_YEAR = 2000;
export const FIRST_MONTH = 1;
export const FIRST_DAY = 1;
export const ADMIN_PANEL = "Admin panel";
