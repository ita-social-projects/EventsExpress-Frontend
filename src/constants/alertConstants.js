export const ALERT_DELAY = 5000;
export const ALERT = "Alert";
