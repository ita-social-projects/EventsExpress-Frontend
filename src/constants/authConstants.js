export const AUTH_SWITCHER_DONT_HAVE_ACCOUNT = "You havenâ€™t got an account?";
export const AUTH_SWITCHER_HAVE_ACCOUNT = "Have an account?";
export const REGISTER = "Register";
export const LOGIN = "Login";
export const SIGN_IN = "Sign in";
export const FORGOT_PASSWORD = "Forgot password?";
export const CLOSE = "Close";
export const CLEAR = "Clear";
export const SUBMIT = "Submit";
export const USE_NEW_PASSWORD_TO_LOGIN = " Please use it to sign in.";
export const EMAIL_PLACEHOLDER = "Enter Email";
export const PASSWORD_PLACEHOLDER = "Enter Password";
export const CONFIRM_PASSWORD_PLACEHOLDER = "Confirm Password";
export const SIGN_UP = "Sign Up";

export const USER_FORGOT_PASSWORD_MESSAGE =
  "If you forgot your password please enter your email address here.";

export const OUR_ACTION_TO_USER_FORGOT_PASSWORD =
  "We will send you new password.";

export const NEW_PASSWORD_SENT_TO_USER_EMAIL =
  "New password sent to your email.";

export const ADD_MORE = "Add more:";
export const SUBSCRIBE = "Subscribe";
export const OR_SIGN_WITH = "Or sign in with:";

export const REGISTRATION_SUCCESSFUL = "Successful registration!";
export const SELECTED_ACTIVITIES = "Selected activities";
