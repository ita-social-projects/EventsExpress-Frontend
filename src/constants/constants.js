// this constant is used to get a token from localStorage
export const jwtStorageKey = "token";

export default jwtStorageKey;
