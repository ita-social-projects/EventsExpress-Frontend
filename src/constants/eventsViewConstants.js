export const eventViewModeMapper = [
  { id: 1, control: `menu-view-mode__matrix`, viewMode: "matrix" },
  { id: 2, control: `menu-view-mode__list`, viewMode: "list" },
  { id: 3, control: `menu-view-mode__slider`, viewMode: "slider" },
];

export const viewModeTypes = {
  MATRIX: "matrix",
  LIST: "list",
  SLIDER: "slider",
};

export const VIEW_MODE_KEY_FOR_LOCAL_STORAGE = "viewModeEvents";
