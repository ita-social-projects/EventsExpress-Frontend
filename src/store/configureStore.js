import { applyMiddleware, combineReducers, compose, createStore } from "redux";
import thunk from "redux-thunk";
import { routerMiddleware } from "react-router-redux";
import rootReducers from "../reducers/root";

export default function configureStore(history, initialState) {
  const middleware = [thunk, routerMiddleware(history)];

  // In development, use the browser's Redux dev tools extension if installed
  const isDevelopment = process.env.NODE_ENV === "development";
  const composeEnhancers =
    isDevelopment &&
    typeof window !== "undefined" &&
    (window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose);
  const rootReducer = combineReducers({
    ...rootReducers,
  });

  return createStore(
    rootReducer,
    initialState,
    composeEnhancers(applyMiddleware(...middleware)),
  );
}
