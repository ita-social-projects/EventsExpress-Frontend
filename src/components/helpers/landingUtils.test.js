import React from "react";
import { Provider } from "react-redux";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import configureStore from "redux-mock-store";
import viewModeSwitcher from "./landingUtils";
import { viewModeTypes } from "../../constants/eventsViewConstants";

const { MATRIX, SLIDER, LIST } = viewModeTypes;

const events = [
  {
    id: "1",
    title: "Vegan For Beginners Online Workshop",
    description:
      "More and more people are interested in vegan/plant-based eating. Some are curious, some want to get their feet wet, and others are ready to come to the V-side! I am here for you all! I offer information to introduce you to what eating vegan is like. This workshop occurs on Zoom and it would be awesome if you turned your camera on!",
    dateFrom: "2022-03-01T10:00:00.0000000Z",
    dateTo: "2022-03-01T18:30:00.0000000Z",
    isReccurent: true,
    maxParticipants: 250,
    frequency: 2,
    periodicity: "Monthly",
    location: "Online",
    isPublic: true,
    isOnlyForAdults: true,
    categories: [
      {
        id: "3",
        name: "Self-education",
        CategoryGroupId: "2",
        countOfUser: 0,
        countOfEvents: 0,
      },
    ],
    organizers: [
      {
        id: "1",
        username: "Dev",
        email: "dev@gmail.com",
        birthday: "2000-06-14T13:30:00.0000000Z",
        rating: 9.6,
        attitude: "Like",
        userStatusEvent: "Approved",
      },
    ],
    members: [
      {
        id: "1",
        username: "Test",
        email: "test@gmail.com",
        birthday: "1983-01-25T10:15:00.0000000Z",
        rating: 5.6,
        attitude: "Like",
        userStatusEvent: "Approved",
      },
    ],
  },
  {
    id: "2",
    title:
      "Football, Finances & Fundraising Workshop - Get top tips to help your club",
    description:
      "The workshop will cover a range of areas including: budgeting, planning, grant funding, sponsorship, fundraising. Attendees will receive advice and ideas from the tutor and be encouraged to share their experiences and views in breakouts rooms.",
    dateFrom: "2022-02-20T13:30:00.0000000Z",
    dateTo: "2022-02-20T15:30:00.0000000Z",
    isReccurent: true,
    maxParticipants: 100,
    frequency: 1,
    periodicity: "Yearly",
    location: "Map",
    isPublic: true,
    isOnlyForAdults: true,
    categories: [
      {
        id: "8",
        name: "Football",
        CategoryGroupId: "3",
        countOfUser: 0,
        countOfEvents: 0,
      },
    ],
    organizers: [
      {
        id: "1",
        username: "Dev",
        email: "dev@gmail.com",
        birthday: "2000-06-14T13:30:00.0000000Z",
        rating: 9.6,
        attitude: "Like",
        userStatusEvent: "Approved",
      },
    ],
    members: [
      {
        id: "1",
        username: "Test",
        email: "test@gmail.com",
        birthday: "1983-01-25T10:15:00.0000000Z",
        rating: 5.6,
        attitude: "Like",
        userStatusEvent: "Approved",
      },
    ],
  },
];

describe("renders events in different view modes without events", () => {
  it("should return slider", () => {
    const tree = renderer.create(viewModeSwitcher([], SLIDER)).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should return matrix", () => {
    const tree = renderer.create(viewModeSwitcher([], MATRIX)).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should return list", () => {
    const tree = renderer.create(viewModeSwitcher([], LIST)).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

const mockStore = configureStore([]);

const store = mockStore({
  user: {
    id: true,
  },
});

const treeWithEvents = viewMode =>
  renderer.create(
    <BrowserRouter>
      <Provider store={store}>{viewModeSwitcher(events, viewMode)}</Provider>,
    </BrowserRouter>,
  );

describe("renders events in different view modes with events", () => {
  it("should return slider", () => {
    const tree = treeWithEvents(SLIDER).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should return matrix", () => {
    const tree = treeWithEvents(MATRIX).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should return list", () => {
    const tree = treeWithEvents(LIST).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
