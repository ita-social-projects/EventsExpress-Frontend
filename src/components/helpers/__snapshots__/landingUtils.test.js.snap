// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders events in different view modes with events should return list 1`] = `
Array [
  <div
    className="card__item"
  >
    <img
      alt="Event_photo"
      src="data:image/svg+xml,%3Csvg%20width%3D%22344%22%20height%3D%22194%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%3E%3Cdefs%3E%3Cpath%20id%3D%22a%22%20d%3D%22M-1%200h344v194H-1z%22%2F%3E%3C%2Fdefs%3E%3Cg%20transform%3D%22translate(1)%22%20fill%3D%22none%22%20fill-rule%3D%22evenodd%22%3E%3Cmask%20id%3D%22b%22%20fill%3D%22%23fff%22%3E%3Cuse%20xlink%3Ahref%3D%22%23a%22%2F%3E%3C%2Fmask%3E%3Cuse%20fill%3D%22%23BDBDBD%22%20xlink%3Ahref%3D%22%23a%22%2F%3E%3Cg%20mask%3D%22url(%23b)%22%3E%3Cpath%20d%3D%22M173.65%2069.238L198.138%2027%20248%20112.878h-49.3c.008.348.011.697.011%201.046%200%2028.915-23.44%2052.356-52.355%2052.356C117.44%20166.28%2094%20142.84%2094%20113.924c0-28.915%2023.44-52.355%2052.356-52.355%2010%200%2019.347%202.804%2027.294%207.669zm0%200l-25.3%2043.64h50.35c-.361-18.478-10.296-34.61-25.05-43.64z%22%20fill%3D%22%23757575%22%2F%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
    />
    <div
      className="card__item_content"
    >
      <h2
        className="card__item_header"
      >
        Vegan For Beginners Online Workshop
      </h2>
      <p
        className="card__item_description"
      >
        More and more people are interested in vegan/plant-based eating. Some are curious, some want to get their feet wet, and others are ready to come to th...
      </p>
      <div
        className="card__item_bottom-row"
      >
        <span
          className="card__item_footer"
        >
          Online
        </span>
        <span
          className="card__item_footer"
        >
          TUE, MAR 1 12:00 PM GMT+02:00
        </span>
        <label
          className="heart__icon add-favorite-list"
          htmlFor="favorite"
        >
          <button
            id="favorite"
            name="favorite"
            onClick={[Function]}
            type="button"
          >
             
          </button>
        </label>
        <label
          className="join-to-event__icon join-to-event-list"
          htmlFor="joinToEvent"
        >
          <button
            id="joinToEvent"
            name="joinToEvent"
            onClick={[Function]}
            type="button"
          >
             
          </button>
        </label>
        <a
          aria-current={null}
          className="bttn"
          href="/home/events/1"
          onClick={[Function]}
        >
          View detail
        </a>
      </div>
    </div>
  </div>,
  <div
    className="card__item"
  >
    <img
      alt="Event_photo"
      src="data:image/svg+xml,%3Csvg%20width%3D%22344%22%20height%3D%22194%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%3E%3Cdefs%3E%3Cpath%20id%3D%22a%22%20d%3D%22M-1%200h344v194H-1z%22%2F%3E%3C%2Fdefs%3E%3Cg%20transform%3D%22translate(1)%22%20fill%3D%22none%22%20fill-rule%3D%22evenodd%22%3E%3Cmask%20id%3D%22b%22%20fill%3D%22%23fff%22%3E%3Cuse%20xlink%3Ahref%3D%22%23a%22%2F%3E%3C%2Fmask%3E%3Cuse%20fill%3D%22%23BDBDBD%22%20xlink%3Ahref%3D%22%23a%22%2F%3E%3Cg%20mask%3D%22url(%23b)%22%3E%3Cpath%20d%3D%22M173.65%2069.238L198.138%2027%20248%20112.878h-49.3c.008.348.011.697.011%201.046%200%2028.915-23.44%2052.356-52.355%2052.356C117.44%20166.28%2094%20142.84%2094%20113.924c0-28.915%2023.44-52.355%2052.356-52.355%2010%200%2019.347%202.804%2027.294%207.669zm0%200l-25.3%2043.64h50.35c-.361-18.478-10.296-34.61-25.05-43.64z%22%20fill%3D%22%23757575%22%2F%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E"
    />
    <div
      className="card__item_content"
    >
      <h2
        className="card__item_header"
      >
        Football, Finances & Fundraising Workshop - Get top tips to help your club
      </h2>
      <p
        className="card__item_description"
      >
        The workshop will cover a range of areas including: budgeting, planning, grant funding, sponsorship, fundraising. Attendees will receive advice and id...
      </p>
      <div
        className="card__item_bottom-row"
      >
        <span
          className="card__item_footer"
        >
          Map
        </span>
        <span
          className="card__item_footer"
        >
          SUN, FEB 20 3:30 PM GMT+02:00
        </span>
        <label
          className="heart__icon add-favorite-list"
          htmlFor="favorite"
        >
          <button
            id="favorite"
            name="favorite"
            onClick={[Function]}
            type="button"
          >
             
          </button>
        </label>
        <label
          className="join-to-event__icon join-to-event-list"
          htmlFor="joinToEvent"
        >
          <button
            id="joinToEvent"
            name="joinToEvent"
            onClick={[Function]}
            type="button"
          >
             
          </button>
        </label>
        <a
          aria-current={null}
          className="bttn"
          href="/home/events/2"
          onClick={[Function]}
        >
          View detail
        </a>
      </div>
    </div>
  </div>,
  ",",
]
`;

exports[`renders events in different view modes with events should return matrix 1`] = `
Array [
  <div
    className="event-matrix"
  >
    <div
      className="event-card"
    >
      <img
        alt="Poster"
        className="event-card__poster"
        src="football-g10e9bda93_1920.jpg"
      />
      <div
        className="event-card__inner-container"
      >
        <p
          className="event-card__inner-text"
        >
          Vegan For Beginners Online Workshop
        </p>
      </div>
      <label
        className="heart__icon add-favorite-matrix"
        htmlFor="favorite"
      >
        <button
          id="favorite"
          name="favorite"
          onClick={[Function]}
          type="button"
        >
           
        </button>
      </label>
      <label
        className="join-to-event__icon join-to-event-matrix"
        htmlFor="joinToEvent"
      >
        <button
          id="joinToEvent"
          name="joinToEvent"
          onClick={[Function]}
          type="button"
        >
           
        </button>
      </label>
    </div>
    <div
      className="event-card"
    >
      <img
        alt="Poster"
        className="event-card__poster"
        src="football-g10e9bda93_1920.jpg"
      />
      <div
        className="event-card__inner-container"
      >
        <p
          className="event-card__inner-text"
        >
          Football, Finances & Fundraising Workshop - Get top tips to help your club
        </p>
      </div>
      <label
        className="heart__icon add-favorite-matrix"
        htmlFor="favorite"
      >
        <button
          id="favorite"
          name="favorite"
          onClick={[Function]}
          type="button"
        >
           
        </button>
      </label>
      <label
        className="join-to-event__icon join-to-event-matrix"
        htmlFor="joinToEvent"
      >
        <button
          id="joinToEvent"
          name="joinToEvent"
          onClick={[Function]}
          type="button"
        >
           
        </button>
      </label>
    </div>
  </div>,
  ",",
]
`;

exports[`renders events in different view modes with events should return slider 1`] = `
Array [
  <div
    className="slick-slider slick-initialized"
  >
    
    <div
      className="slick-list"
    >
      <div
        className="slick-track"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseOver={[Function]}
        style={
          Object {
            "WebkitTransform": "translate3d(0px, 0px, 0px)",
            "WebkitTransition": "",
            "msTransform": "translateX(0px)",
            "opacity": 1,
            "transform": "translate3d(0px, 0px, 0px)",
            "transition": "",
          }
        }
      >
        <div
          aria-hidden={false}
          className="slick-slide slick-active slick-center slick-current"
          data-index={0}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="event-card"
            >
              <img
                alt="Poster"
                className="event-card__poster"
                src="football-g10e9bda93_1920.jpg"
              />
              <div
                className="event-card__inner-container"
              >
                <p
                  className="event-card__inner-text"
                >
                  Vegan For Beginners Online Workshop
                </p>
              </div>
              <label
                className="heart__icon add-favorite-slider"
                htmlFor="favorite"
              >
                <button
                  id="favorite"
                  name="favorite"
                  onClick={[Function]}
                  type="button"
                >
                   
                </button>
              </label>
              <label
                className="join-to-event__icon join-to-event-slider"
                htmlFor="joinToEvent"
              >
                <button
                  id="joinToEvent"
                  name="joinToEvent"
                  onClick={[Function]}
                  type="button"
                >
                   
                </button>
              </label>
            </div>
          </div>
        </div>
        <div
          aria-hidden={false}
          className="slick-slide slick-active"
          data-index={1}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="event-card"
            >
              <img
                alt="Poster"
                className="event-card__poster"
                src="football-g10e9bda93_1920.jpg"
              />
              <div
                className="event-card__inner-container"
              >
                <p
                  className="event-card__inner-text"
                >
                  Football, Finances & Fundraising Workshop - Get top tips to help your club
                </p>
              </div>
              <label
                className="heart__icon add-favorite-slider"
                htmlFor="favorite"
              >
                <button
                  id="favorite"
                  name="favorite"
                  onClick={[Function]}
                  type="button"
                >
                   
                </button>
              </label>
              <label
                className="join-to-event__icon join-to-event-slider"
                htmlFor="joinToEvent"
              >
                <button
                  id="joinToEvent"
                  name="joinToEvent"
                  onClick={[Function]}
                  type="button"
                >
                   
                </button>
              </label>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-center slick-cloned"
          data-index={2}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="event-card"
            >
              <img
                alt="Poster"
                className="event-card__poster"
                src="football-g10e9bda93_1920.jpg"
              />
              <div
                className="event-card__inner-container"
              >
                <p
                  className="event-card__inner-text"
                >
                  Vegan For Beginners Online Workshop
                </p>
              </div>
              <label
                className="heart__icon add-favorite-slider"
                htmlFor="favorite"
              >
                <button
                  id="favorite"
                  name="favorite"
                  onClick={[Function]}
                  type="button"
                >
                   
                </button>
              </label>
              <label
                className="join-to-event__icon join-to-event-slider"
                htmlFor="joinToEvent"
              >
                <button
                  id="joinToEvent"
                  name="joinToEvent"
                  onClick={[Function]}
                  type="button"
                >
                   
                </button>
              </label>
            </div>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={3}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <div
              className="event-card"
            >
              <img
                alt="Poster"
                className="event-card__poster"
                src="football-g10e9bda93_1920.jpg"
              />
              <div
                className="event-card__inner-container"
              >
                <p
                  className="event-card__inner-text"
                >
                  Football, Finances & Fundraising Workshop - Get top tips to help your club
                </p>
              </div>
              <label
                className="heart__icon add-favorite-slider"
                htmlFor="favorite"
              >
                <button
                  id="favorite"
                  name="favorite"
                  onClick={[Function]}
                  type="button"
                >
                   
                </button>
              </label>
              <label
                className="join-to-event__icon join-to-event-slider"
                htmlFor="joinToEvent"
              >
                <button
                  id="joinToEvent"
                  name="joinToEvent"
                  onClick={[Function]}
                  type="button"
                >
                   
                </button>
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
    
    
  </div>,
  ",",
]
`;

exports[`renders events in different view modes without events should return list 1`] = `null`;

exports[`renders events in different view modes without events should return matrix 1`] = `
<div
  className="event-matrix"
/>
`;

exports[`renders events in different view modes without events should return slider 1`] = `
<div
  className="slick-slider slick-initialized"
>
  
  <div
    className="slick-list"
  >
    <div
      className="slick-track"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onMouseOver={[Function]}
      style={
        Object {
          "WebkitTransform": "translate3d(0px, 0px, 0px)",
          "WebkitTransition": "",
          "msTransform": "translateX(0px)",
          "opacity": 1,
          "transform": "translate3d(0px, 0px, 0px)",
          "transition": "",
        }
      }
    />
  </div>
  
  
</div>
`;
